.github/workflows/ci.yaml

```diff
@@ -66,21 +37,10 @@ jobs:
         uses: charmed-kubernetes/actions-operator@main
         with:
           provider: microk8s
-        # These two steps (download and load image) are needed until the OCI image is published to ROCKS.
-      - name: Download image built in build oci image step
-        uses: actions/download-artifact@v2
-        with:
-          name: image
-          path: /tmp
-      - name: Load Image into microk8s
-        run: |
-          /usr/bin/sg microk8s -c "microk8s ctr image import /tmp/image.tar"
       - name: Run integration tests
-        # set a predictable model name so it can be consumed by charm-logdump-action
-        run: tox -e integration -- --model testing
```

Comment:
seems like  charm-logdump-action still exists so might want to add `--model testing` back in